File: service/hospital-service/src/main/java/com/simp/service/hospital/HospitalServiceApplication.java
package com.simp.service.hospital;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.cloud.openfeign.EnableFeignClients;
import org.springframework.web.reactive.config.EnableWebFlux;
import reactivefeign.spring.config.EnableReactiveFeignClients;

@EnableFeignClients(basePackages = {"com.simp"})
@SpringBootApplication(scanBasePackages = {"com.simp"})
@ConfigurationPropertiesScan("com.simp")
@EnableWebFlux
@EnableReactiveFeignClients(basePackages = {"com.simp"})
public class HospitalServiceApplication {
    public static void main(String[] args) {
        SpringApplication.run(HospitalServiceApplication.class, args);
    }
}

File: service/hospital-service/src/main/kotlin/com/simp/service/hospital/HospitalServiceApplication.kt
package com.simp.service.hospital

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.cloud.openfeign.EnableFeignClients

@EnableFeignClients(basePackages = ["com.simp"])
@SpringBootApplication(scanBasePackages = ["com.simp"])
@ConfigurationPropertiesScan("com.simp")
class HospitalServiceApplication

fun main(args: Array<String>) {
    runApplication<HospitalServiceApplication>(*args)
}
